tools/st2-setup-samples.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+PACK=$1
+if [[ "$PACK" == "examples" ]]; then
+    PATH="/usr/share/doc/st2/"
+elif [[ "$PACK" == "tests" ]]; then
+    PATH="/usr/share/stackstorm/"
+fi
+
+echo -e "Adding packs_base_paths for ${PATH}/${PACK}..."
+grep packs_base_path /etc/st2/st2.conf || echo -e "\n[content]\npacks_base_paths = ${PATH}/${PACK}" >> /etc/st2/st2.conf'
```

Comment:
I know this is not an issue in this case, but there is a double slash between path and pack.